### ISSUE TITLE
Can't create BREAD relation

### ISSUE THREAD

**Issue by anthonykozak** [thedevdojo/voyager#2023]
- Laravel Version: 5.5.14
- Voyager Version: 1.0.5
- PHP Version: 7.1.7
- Database Driver & Version: Mysql 5.7.19

### Description:
Could not create a BREAD relationship.

### Steps To Reproduce:
**-Create a "tests" table with Voyager, and check "generate Model", Or create the table in phpMyAdmin and create the model with Artisan
-Generated model in app/ is** 
```
<?php

namespace App;

use Illuminate\Database\Eloquent\Model;

class Test extends Model
{
    //
}
```
**-Create the BREAD for the tests table**
![bug_1](https://user-images.githubusercontent.com/973985/31587086-1c954c9a-b1a9-11e7-975f-bebf6b216597.jpg)

**-Create any relationship**
![screenshot-2017-10-15 edit bread for tests table](https://user-images.githubusercontent.com/973985/31587088-2473c234-b1a9-11e7-8aa2-fdaac6cf1c57.png)
**-Got the error :**
![fireshot capture 18 - edit bread for tests table - http___test dev_admin_database_tests_bread_edit](https://user-images.githubusercontent.com/973985/31587083-16cc3828-b1a9-11e7-8fec-066153077d7b.jpg)

**Comment by lablushah** [comment]
Your namespace field is blank. Try to fill it with App\Test. Hope it will solve the problem.

**Comment by anthonykozak** [comment]
Nice ! So the namespace field should be required I guess ;)
Thank you so much!

**Comment by fletch3555** [comment]
You're correct, it should be required.  We don't have any validation on that field because a bit of common sense should dictate it be required.  Honest mistake of course, we'll get some validation added in, or someone else can with a PR.

**Comment by kangketikonlen** [comment]
Hello, did you manage to resolve this?
The models doesn't generate anything.
![Screen Shot 2020-02-24 at 3 07 40 PM](https://user-images.githubusercontent.com/17847417/75136484-6f379780-5717-11ea-8718-3f259e547361.png)
and it didn't show anything in my dropdown list.
![Screen Shot 2020-02-24 at 3 05 33 PM](https://user-images.githubusercontent.com/17847417/75136518-8f675680-5717-11ea-84ab-399db914b931.png)

Did I miss something here?

**Comment by MrCrayon** [comment]
@AkasakaRyu this issue can be resolved as explained in previous comments.

Your problem doesn't look related to this issue. 

If you have a question please ask on Slack chat.

**Comment by fletch3555** [comment]
Also, this issue was closed nearly 2.5 years ago, and was for a MUCH older version of Voyager than you're likely using.  Many things have changed since then, especially around how relationships work, so their solution is likely unrelated to your issue.

Please dont comment on such old closed issues.  Instead you can ask questions in slack or report bugs in a new issue.
